skimage/morphology/tests/test_misc.py

```diff
@@ -138,7 +172,7 @@ def test_one_connectivity_holes():
         ],
         bool,
     )
-    observed = remove_small_holes(test_holes_image, area_threshold=3)
+    observed = remove_small_holes(test_holes_image, max_size=3)
```

Comment:
How come isn't it
```suggestion
    observed = remove_small_holes(test_holes_image, max_size=2)
```
?